app/templates/administrative-units/administrative-unit.hbs

```diff
@@ -30,15 +30,15 @@
             </AuNavigationNestedLink>
           </li>
         {{/if}}
-        {{#unless this.isAgbOrApb}}
+        {{#if (and (not this.isAgbOrApb) (not this.isIgs))}}
```

Comment:
What's the reason of using the if here instead of the `unless (or this.isAgbOrApb this.isIgs)

Reply:
Ember complains if you put a `not` in an `unless` :sweat_smile: I don't remember the exact warning message though.